### ISSUE TITLE
modify CNN to use any PyTorch LR Scheduler

### ISSUE THREAD

**Issue by sammlapp** [kitzeslab/opensoundscape#910]
currently only uses StepLR with configurable update interval and ratio

should simply allow user to set an initialized LR Scheduler object as cnn.lr_scheduler or something
